Header header

uint8 JOYSTICK=0
uint8 PLANNER=1
uint8 LANDING=2

uint8 state
